terraform/modules/vpc-hub/locals.tf

```diff
@@ -0,0 +1,227 @@
+locals {
+  # 1000 = intra-vpc traffic
+  # 2000 = inter-vpc traffic (dynamic rules)
+  # 3000 = deny east-west
+  # 4000 = private address ranges
+  # 5000 = internet access
+  # 6000 = public address ranges (dynamic)
+  # 7000 = access from internet
+
+  static_acl_rules = {
+    allow_vpc_cidr_in = {
+      cidr_block  = var.vpc_cidr
+      egress      = false
+      from_port   = null
+      protocol    = "-1"
+      rule_action = "allow"
+      rule_number = 1000
+      to_port     = null
+    },
+    allow_vpc_cidr_out = {
+      cidr_block  = var.vpc_cidr
+      egress      = true
+      from_port   = null
+      protocol    = "-1"
+      rule_action = "allow"
+      rule_number = 1000
+      to_port     = null
+    },
+    allow_10-0-0-0_in = {
```

Comment:
Are the network firewall rules in place that prevent hopping between vpcs?

Reply:
Yes and no. Traffic inside a "domain" like prod/preprod and dev/test hits the MP Transit Gateway and goes directly to the destination VPC.
EG. `laa-test` can send to `platforms-development` without being passed through the `core-network-services["external-inspection"]` VPC.
EG. `laa-development` can send to `core-network-services["non_live_data"]` without being passed through the `core-network-services["external-inspection"]` VPC.

When traffic wants to move between those domains - `laa-development` to `laa-production` or `core-shared-services["live_data"]` then it would move through the MP Network firewall in the `core-network-services["external-inspection"]` VPC.